examples/trackML/src/SimpleStepper.cxx

```diff
@@ -1,10 +1,13 @@
 #include "SimpleStepper.h"
-
 #include <management/GeoManager.h>
 #include <volumes/Tube.h>
 #include "Track.h"
 #include "HelixPropagator.h"
 #include "vectorFlow/SystemOfUnits.h"
+#include "Geant/ConstFieldHelixStepper.h"
+#include "Tracks_v.h"
+
+extern int gNkills;
```

Comment:
I do not really like passing configuration parameters as external globals. Rather make a data member, configuration object, ... that you can setup at initialisation time.

Reply:
Solved. Removed external global variable...